.github/workflows/build_wheels.yml

```diff
@@ -89,24 +89,3 @@ jobs:
       - uses: actions/upload-artifact@v3
         with:
           path: dist/*.tar.gz
-
-  upload_pypi:
-    needs: [build_wheels, build_sdist]
-    runs-on: ubuntu-latest
-    environment:
-      name: pypi
-      url: https://pypi.org/p/asv_runner
-    permissions:
-      id-token: write  # for trusted publishing
-    # upload to PyPI on every tag starting with 'v'
-    if: github.event_name == 'push' && startsWith(github.ref, 'refs/tags/v') && github.repository == 'airspeed-velocity/asv'
-    # alternatively, to publish when a GitHub Release is created, use the following rule:
-    # if: github.event_name == 'release' && github.event.action == 'published'
-    steps:
-      - uses: actions/download-artifact@v3
-        with:
-          # unpacks default artifact into dist/
-          # if `name: artifact` is omitted, the action will create extra parent dir
-          name: artifact
-          path: dist
-      - uses: pypa/gh-action-pypi-publish@release/v1
```

Comment:
Are you doing this differently now?

Reply:
We never set up trusted publishing so this wasn't running, and shouldn't have been here. The release notes clarify this, explaining that we use `twine` for manual management.

I prefer manual management since you can test the wheels before uploading, the trusted publisher workflow means the only way to fix a bug is either a new release or a yank.